src/components/Tabs/Tabs.js

```diff
@@ -97,13 +103,13 @@ export default class Tabs extends React.Component {
     });
   };
 
-  selectTabAt = index => {
+  selectTabAt = (index, onSelectionChange) => {
     if (this.state.selected !== index) {
       this.setState({
         selected: index,
       });
-      if (this.props.onSelectionChange) {
-        this.props.onSelectionChange(index);
+      if (onSelectionChange !== undefined) {
```

Comment:
Just a little thing - If we’d do more strict check than is-truthy, I’d recommend `typeof f === 'function'`.

Reply:
Good idea. I'll switch out the conditional with the recommended check.